### PR TITLE
Notification multisig

### LINKED ISSUE #751: Add integrated service for multisig + cosigner ws listener

### PR DESCRIPTION
**Pull request by rg911**
- Subscribe to multisig account from cosigner 
- Fixed #751

### CODE REVIEW

File: src/infrastructure/Listener.ts [hunk 1]
```diff
@@ -92,9 +94,14 @@ export class Listener implements IListener {
...
          */
         private websocketInjected?: any,
+        /**
+         * Multisig repository for resolving multisig accounts
+         */
+        private multisigRepository?: MultisigRepository,
     ) {
         this.url = url.replace(/\/$/, '');
         this.messageSubject = new Subject();
+        this.multisigRepository = this.multisigRepository ? this.multisigRepository : new MultisigHttp(this.namespaceRepository.getUrl());
```
Comment (fboucquez) [inline src/infrastructure/Listener.ts:104]: Why asking for the url to the node repository? The URL is up there plus I would like to keep the repository URL agnostic.

Comment (fboucquez) [reply src/infrastructure/Listener.ts:97]: It's kind of hacky. I would make multisigRepository required like namespaceRepository and remove getUrl from the namespace repository. If somebody is already using the ListnerHttp, this person would know how to create a multisigRepository.

**Comment by fboucquez** [comment]
@rg911 , there is channel named 'modifyMultisigAccount'. But unused? are we missing listeners?

File: src/infrastructure/Listener.ts (hunk 1 above)
Comment (rg911) [reply src/infrastructure/Listener.ts:97]: is the way not to introduce a breaking change at this point of time

Comment (fboucquez) [reply src/infrastructure/Listener.ts:97]: getUrl is breaking the abstraction. Also why in the NamespaceRepository and not in the other ones (I'm not saying they should have it, it's just inconsistent). 

If the devs coded the clients correctly, the construction of the Listener should be behind the scenes without breaking the code. If not, most people would know how to create the repositories used by the listener.

Those are my points, if you like to merge it's all good.

**Pull request merged**